docs/integrations/saas-cloud/miro.md

```diff
@@ -0,0 +1,109 @@
+    2. **Folder Name**. You can retain the existing name, or enter a name of your choice for the app.�
+    3. Select the **Location in Library** (the default is the **Personal** folder in the library), or click **New Folder** to add a new folder.
+1. Click **Next**.
+
+Once an app is installed, it will appear in your **Personal** folder, or other folder that you specified. From here, you can share it with your organization.
+
+Panels will start to fill automatically. It's important to note that each panel slowly fills with data that matches the time range query and has been received since the panel was created. Results will not be available right away, but with a bit of time, you'll see full graphs and maps.
+
+## Viewing Miro Dashboards​
+
+### Miro - Overview
+
+The **Miro - Overview** provides a comprehensive overview of events related to Miro. The dashboard features various widgets, including Total Events, Total Board Events, Distribution of App Events, Events By Team, Boards Created and Opened over Time, Events Over Time, Geo-Location of Events, Recent File Event Summary, and Recent Event Summary.
+
+The widgets allow users to track and analyze event-related information, including the total number of events, their distribution, team-wise breakdown, creation and opening of boards over time, and their geographical locations. The Recent File Event Summary and Recent Event Summary widgets provide a summary of the latest events for quick reference. A dashboard is a useful tool for monitoring and improving event management by providing quick access to relevant information.
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Miro/Miro-Overview.png' alt="Miro-Overview.png" />
+
+### Miro - Security Activity
+
+The **Miro - Security Activity** provides a comprehensive overview of user security events related to Miro. The dashboard features various widgets, including Total SignIn Security Events, Total Account(s) Created, Total Account(s) Deleted, Total User(s) Deactivated, Total User(s) Reactivated, Distribution of SignIn Security Events, Sign-In over time by Authentication Methods, and SignIn from Risky Geo-Locations.
+
+The widgets allow users to track and analyze user security events, including the total number of sign-in security events, account creation, deletion, deactivation, and reactivation. The Distribution of SignIn Security Events widget provides an overview of sign-in patterns across different events, while the SignIn over time by Authentication Methods widget offers a breakdown of sign-ins by the authentication method. The SignIn from Risky Geo-Locations widget helps identify geographic locations that pose a higher risk to user security. A dashboard is a useful tool for monitoring and improving user security by providing quick access to relevant information.
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Miro/Miro-Security-Activity.png' alt="Miro-Security-Activity.png" />
```

Comment:
@JV0812 please add this Dashboard image to AWS S3 as we discussed in Asana. Otherwise it won't show here or in the UI.

Reply:
App is still under QA, once it ready i will add the image to S3 bucket.